src/constants/strings.js

```diff
@@ -0,0 +1,74 @@
+
+export const memberProfiles = [{
```

Comment:
This file name is very unhelpful. If I'm navigating the code base constants/strings is not a useful filename. Also, not everything is a string in here. memberProfiles that you're exporting is literally an object.